src/orange/detail/UnitInserter.cc

```diff
@@ -191,6 +191,19 @@ UnitInserter::UnitInserter(UniverseInserter* insert_universe, Data* orange_data)
     // Initialize scalars
     orange_data_->scalars.max_faces = 1;
     orange_data_->scalars.max_intersections = 1;
+
+    // Bounding box bumper and converter *to* fast real type *from* regular
+    // real type: conservatively expand to twice the potential bump distance
+    // from a boundary so that the bbox will enclose the point even after a
+    // potential bump
+    calc_bumped_ = BoundingBoxBumper<fast_real_type, real_type>(
+        [&tol = orange_data_->scalars.tol] {
+            Tolerance<real_type> bbox_tol;
+            bbox_tol.rel = 2 * tol.rel;
+            bbox_tol.abs = 2 * tol.abs;
+            CELER_ENSURE(bbox_tol);
+            return bbox_tol;
+        }());
```

Comment:
Better to put this as a member data initializer rather than assigning it later; you could turn the lambda into an anonymous-namespace helper function to make it a bit cleaner.